fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          users: data,
+          isLoading: false
+        });
+      })
+      .catch(err => {
+        if (err) {
+          console.error(err);
+        }
+      });
```

Comment:
Generally speaking, `err` should always be truthy if the `.catch()` callback gets invoked.